examples/crowdfunding/test/happypaths.js

```diff
@@ -0,0 +1,396 @@
+import {
```

Comment:
Please update this test to use timestamp, you can refer to failing-paths PR.

Reply:
I am already using `runtime.setRoundAndTimeStamp` in these tests. Since user has the control, i am using simple numbers to represent `timestamp`. Or did you mean something else?
eg. `runtime.setRoundAndTimestamp(2, 5);` (setting current `ts` to 5, and startTs, endTs to `1, 10` respectively).
Will update if you think we should use actual timestamps instead.

Reply:
you have uncomment the lines in teal files, so setting timestamp, rounds has no effect unless tested.

Reply:
ah, updating .teal, yes.